content-templates/default-index.yml

```diff
@@ -1,33 +1,32 @@
-  title: # user input: module title for browser tab and search results
-  description: # user input: a description for site search and SEO
-  ms.date: {{msDate}} # stub with today's date
-  author: {{githubUsername}} # stub with GitHub ID from user settings
-  ms.author: {{msUser}} # stub with Microsoft alias from user settings
-  ms.topic: interactive-tutorial # do we need ms.topic?
-title: {{moduleTitle}} # stubbed from user input in VS Code (https://review.docs.microsoft.com/help/learn/id-guidance-title?branch=master)
-summary: # user input: introductory summary (https://review.docs.microsoft.com/help/learn/id-guidance-introductory-summaries?branch=master)
-abstract: | # user input: learning objectives (https://review.docs.microsoft.com/help/learn/id-guidance-learning-objectives?branch=master)
+  moduleType: {{patternType}}
+  title: {{moduleTitle}}
+  description: TODO This field is not visible when published, but is critical for search engine optimization. Use 2-3 complete, grammatically correct sentences to describe the unit, and include lots of relevant search keywords.
+  ms.date: {{msDate}}
+  author: {{githubUsername}}
+  ms.author: {{msUser}}
+  ms.topic: interactive-tutorial
+title: {{moduleTitle}}
+summary: "TODO This is the 'summary' field in index.yml. 2 sentences, 20-35 words. Don't teach or motivate, avoid using 'learn', and don't duplicate the title. See the guidance [here](https://review.docs.microsoft.com/help/learn/id-guidance-introductory-summaries?branch=master)."
+abstract: |
   By the end of this module, you'll be able to: 
-    - # objective 1
-    - # objective 2
-    - # objective 3 (if needed)
-prerequisites: | # user input: prerequisites (https://review.docs.microsoft.com/help/learn/id-guidance-prerequisites?branch=master)
-  - # prerequisite 1
-  - # prerequisite 2
-  - # prerequisite 3
+    - TODO objective 1
+    - TODO objective 2
+    - "TODO objective 3 (if needed. See guidance [here](https://review.docs.microsoft.com/help/learn/id-guidance-learning-objectives?branch=master).)"
+prerequisites: |
+  - TODO prerequisite 1
+  - TODO prerequisite 2
+  - "TODO prerequisite 3 (if needed; minimum 2 required. See guidance [here](https://review.docs.microsoft.com/help/learn/id-guidance-prerequisites?branch=master))"
 iconUrl: https://docs.microsoft.com/media/learn/module.svg
-levels: # user input: add levels from level taxonomy (https://review.docs.microsoft.com/help/contribute/metadata-taxonomies?branch=master#level)
-  - # level
-roles: # user input: add roles from role taxonomy (https://review.docs.microsoft.com/help/contribute/metadata-taxonomies?branch=master#role)
-  - # role 1
-  - # role 2
-products: # stub default from user settings; user input: add product from product taxonomy (https://review.docs.microsoft.com/help/contribute/metadata-taxonomies?branch=master#product)
+levels:
+  - beginner
+roles:
+  - developer
```

Comment:
Added valid values here avoid build breakage. In my experience, people will see these values rendered in the preview and fix them (or ask how to fix them) if they're not appropriate for the content, but with a broken build they're blocked and frustrated.

Reply:
I agree with the "make it build" philosophy.

Our choice is between break-the-build values or potentially-incorrect values

We made the conservative choice thinking the scaffold/templates are most valuable for new authors and we didn't want to mislead them.